File: day8/python/test_solution.py
import unittest
from solution import solve_part1

class TestDay8(unittest.TestCase):
    def test_part1(self):
        with open('input_test.txt', 'r') as f:
            content = f.read()
        self.assertEqual(solve_part1(content), 40)

if __name__ == '__main__':
    unittest.main()

File: day8/kotlin/test_solution.kt
import java.io.File

fun main() {
    val content = File("input_test.txt").readText().trim()
    val result = solvePart1(content)
    println("Testing Part 1: Expected 40, Got $result")
    assert(result == 40) { "Test failed: Expected 40, Got $result" }
    println("Test passed!")
}
